### PR TITLE
[CHI-281] Create an alias to run all the commands

### PR DESCRIPTION
**Pull request by HectorGarciaAlvarez**
@jllr @dani-cl-madrid @juanmorra please review

### CODE REVIEW

**Comment by jllr** [comment]
@HectorGarciaAlvarez we should include the way to create the alias before use chi start i in getting started page

File: src/website/views/getting-started/index.pug [hunk 1]
```diff
@@ -36,7 +36,14 @@ p.-text
 
 .-mb--2
   :code(lang='bash')
-    $ scripts/docker.sh start
+    scripts/docker.sh alias
+    source ~/.bash_profile
+
+  | Now your environment it's set up. To start development run:
```
Comment (jllr) [inline src/website/views/getting-started/index.pug]: @HectorGarciaAlvarez this one should render as a paragraph in 14px. Right now, it is included inside the code block and is rendering at 16px. Please add p.-text and remove tabulation so it is indented properly

**Comment by jllr** [comment]
@mattnickles any suggestions in the copy?

File: src/website/views/getting-started/index.pug (hunk 1 above)
Comment (dani-cl-madrid) [inline src/website/views/getting-started/index.pug]:
```suggestion
p.-text Now your environment it's set up. To start development run:
```

**Comment by HectorGarciaAlvarez** [comment]
test this please

**Pull request merged**